components/formats-bsd/src/loci/formats/in/TiffReader.java

```diff
@@ -180,7 +181,15 @@ protected void initStandardMetadata() throws FormatException, IOException {
 
     // check for ImageJ-style TIFF comment
     boolean ij = checkCommentImageJ(comment);
-    if (ij) parseCommentImageJ(comment);
+    if (ij) {
+      parseCommentImageJ(comment);
+    }
+    else {
+      ij = checkCommentImageJ(lastComment);
+      if (ij) {
+        parseCommentImageJ(lastComment);
```

Comment:
The logic looks correct, but the reuse of `ij` seems unnecessary.  Why not simply:

```
if (checkCommentImageJ(comment)) {
  parseCommentImageJ(comment);
}
else if (checkCommentImageJ(lastComment)) {
  parseCommentImageJ(lastComment);
}
```

?

Reply:
```ij``` is used and reset for the benefit of this check: https://github.com/melissalinkert/bioformats/blob/b03f5dadf6a6f71f7c21292b704466a3f71ea4c3/components/formats-bsd/src/loci/formats/in/TiffReader.java#L202